src/implementations/policies/random_policy.jl

```diff
@@ -1,4 +1,4 @@
-export RandomPolicy, RandomStartPolicy
+export RandomPolicy
```

Comment:
For chance nodes, `get_prob(::RandomPolicy, env)` should return `[a.prob for a in get_legal_actions(env)]`